docs/source/modules/connections.rst

```diff
@@ -7,4 +7,5 @@ Connection Object
    :members:
    :exclude-members: DataError, DatabaseError, Error, InterfaceError,
                      IntegrityError, InternalError, NotSupportedError,
-                     OperationalError, ProgrammingError, Warning
+		     OperationalError, ProgrammingError, Warning,
+		     escape, literal, write_packet
```

Comment:
Now that I think about it, it's a`.rst` file so autopep8 wouldn't do anything. That's probably why it ended up that way because autopep8 didn't kick in like I expected.